data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/resistance/resistance_affect_self_no.cfg

```diff
@@ -0,0 +1,31 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [resistance]affect_self=
+##
+# Actions:
+# Give all units 50% resistance to all damage types
```

Comment:
Give all units an ability would give resistance to damage, but which doesn't affect any units

Reply:
Updated.